content/en/network_monitoring/devices/syslog.md

```diff
@@ -0,0 +1,198 @@
+   * **`service`**: Optional service name for unified service tagging.
+
+3. [Restart the Datadog Agent][101] to apply the changes.
+
+   ```
+   sudo systemctl restart datadog-agent
+   ``` 
+
+[101]: /agent/configuration/agent-commands/#restart-the-agent
+
+{{% /tab %}}
+{{% tab "Windows" %}}
+
+1. Ensure the following settings are enabled in your `C:\ProgramData\Datadog\datadog.yaml` file:
+
+   ```
+   logs_enabled: true # enable logs collection
+   logs_config:
+   use_sourcehost_tag: true # adds a source_host tags to logs with the source IP
+   ```
+
+2. Create a Syslog listener configuration:
+
+   - In `C:\ProgramData\Datadog\conf.d\`, create a directory called `syslog.d/`.
+   - Inside `syslog.d/`, create a file named `conf.yaml` with the following:
+
+   ```yaml
+   logs:
+     - type: syslog
+       port: 514
+       protocol: udp
+       source: syslog
+       service: <service> # optional tag
+   ```
+
+   * **`port`**: The port the Agent listens on for Syslog messages. Default is 514.
+   * **`protocol`**: Set to `udp` or `tcp` based on your device configuration.
+   * **`source`**: Custom source name for these logs in Datadog. Use `syslog` to correlate with NDM devices.
+   * **`service`**: Optional service name for unified service tagging.
+
+3. [Restart the Datadog Agent][101] service from the Services console to apply the changes.
+
+[101]: /agent/configuration/agent-commands/#restart-the-agent
+
+{{% /tab %}}
+{{% tab "Docker" %}}
+
+1. Ensure the following settings are enabled in your `/etc/datadog-agent/datadog.yaml` file in your Docker volume:
+
+   ```yaml
+   logs_enabled: true # enable logs collection
+   logs_config:
+   use_sourcehost_tag: true # adds a source_host tags to logs with the source IP
+   ```
+
+2. Create a Syslog listener configuration:
+
+   - Mount a volume to `/etc/datadog-agent/conf.d/` and create a directory called `syslog.d/`.
+   - Inside `syslog.d/`, create a file named `conf.yaml` with the following:
+
+   ```yaml
+   logs:
+     - type: syslog
+       port: 514
+       protocol: udp
+       source: syslog
+       service: <service> # optional tag
+   ```
+
+   * **`port`**: The port the Agent listens on for Syslog messages. Default is 514.
+   * **`protocol`**: Set to `udp` or `tcp` based on your device configuration.
+   * **`source`**: Custom source name for these logs in Datadog. Use `syslog` to correlate with NDM devices.
+   * **`service`**: Optional service name for unified service tagging.
+
+3. Restart the [Docker container][101] to apply the changes.
+
+[101]: /agent/configuration/agent-commands/#restart-the-agent
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Log parsing
+
+The above steps send your Syslog data to Datadog. To ensure NDM correctly associates these logs with your monitored network devices, set up a custom log parsing pipeline that populates the `syslog_ip` tag with each device's source IP address.
```

Comment:
Not sure if this is fully technically correct - I was just a little concerned that users might misunderstand and think that the config steps caused a one-time migration of data to Datadog rather than an ongoing pipeline.
```suggestion
After you complete the above steps, Syslog can start sending log data to Datadog. To ensure NDM correctly associates these logs with your monitored network devices, set up a custom log parsing pipeline that populates the `syslog_ip` tag with each device's source IP address.
```